kayak_core/src/widget_manager.rs

```diff
@@ -384,4 +385,97 @@ impl WidgetManager {
         }
         None
     }
+
+    /// Get the nodes under the given position
+    ///
+    /// The first element in the returned tuple is the node deemed as the "best match". A node is a best match when
+    /// it is the deepest matching node in the hierarchy or it has a higher z-index than the other matches.
+    ///
+    /// # Arguments
+    ///
+    /// * `position`: The 2D point to check under
+    /// * `parent`: The parent to start at (or `None` for the root element)
+    /// * `respect_pointer_events`: Whether a widget's `pointer_events` style should be respected
```

Comment:
Not a huge fan of adding this parameter. Another reason to maybe move this logic to `KayakContext::process_events`.